### PR TITLE
Update dependencies & plugins

### PR DESCRIPTION
**Pull request by wendigo**

### CODE REVIEW

**Comment by wendigo** [comment]
@martint @electrum ptal

File: CHANGES.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ Airbase 157
   - error_prone_annotations 2.27.1 (from 2.27.0)
   - guava 33.2.1-jre (from 33.1.0-jre)
   - jackson 2.17.1 (from 2.17.0)
+  - kotlin 2.0.0 (from 1.9.23)
```
Comment (findepi) [inline CHANGES.md:11]: is it a safe change?

File: CHANGES.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ Airbase 157
   - maven-build-helper 3.6.0 (from 3.5.0)
   - maven-license-plugin 4.5 (from 4.3)
   - maven-modernizer-plugin 2.9.0 (from 2.7.0)
+  - maven-sortpom-plugin 4.0.0 (from 3.4.1)
```
Comment (findepi) [inline CHANGES.md:18]: is it a safe change?

File: airbase/pom.xml [hunk 3]
```diff
@@ -969,6 +969,7 @@
                         <gitDescribe>
                             <tags>true</tags>
                         </gitDescribe>
+                        <injectAllReactorProjects>true</injectAllReactorProjects>
```
Comment (findepi) [inline airbase/pom.xml:972]: thank you!
can you please move this to a separate PR that I can approve?
And add CHANGES entry.

File: CHANGES.md (hunk 2 above)
Comment (wendigo) [reply CHANGES.md:18]: It is. Validated on Airlift and Trino.

File: CHANGES.md (hunk 1 above)
Comment (wendigo) [reply CHANGES.md:11]: Yes. It's basically a "compiler is now stable" release.

File: airbase/pom.xml (hunk 3 above)
Comment (wendigo) [reply airbase/pom.xml:972]: https://github.com/airlift/airbase/pull/412

**Comment by wendigo** [comment]
@electrum PTAL

**Review by electrum — APPROVED** [review]

**Pull request merged**